installer/parse-install-yaml.py

```diff
@@ -88,6 +88,20 @@ def main():
                 else:
                     return show_error("Unknown ROS install type: '{0}'".format(source_type))
 
+            # Non ros targets that are packaged to be built with catkin
```

Comment:
I get why `catkin` is a better description than `ros` for these kind of cmake packages.
But shouldn't the handling be exactly the same as a ROS target?

These cmake packages can actually have their own dependencies that have to be resolved as well. So the `package.xml` should be read and handled.

Reply:
There is only one difference and that is system install. These cmake packages won't follow the rules of ROS packages. It is possible to use the same if block as that of "ros" but it would add another if block checking if the type is catkin, skip system install check. That becomes a lot of nested if else blocks

Reply:
Than it would be nicer to grasp the common part in a function maybe?
This would add the dependency resolving for free to the catkin part.